interface/portraits/portraits_highelf — копия.gfx

```diff
@@ -0,0 +1,406 @@
+# hair, mouth, chin, nose, eyes, cheek, ear, neck
```

Comment:
Sooo, what is this file supposed to do?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81070065-cd40e200-8ef3-11ea-98bc-d4716392ef43.png)

</details>